Source/SwiftLintFramework/Rules/Lint/EmptyMethodOverrideRule.swift

```diff
@@ -0,0 +1,158 @@
+import Foundation
+import SourceKittenFramework
+
+public struct EmptyMethodOverrideRule: SubstitutionCorrectableASTRule,
+ConfigurationProviderRule, AutomaticTestableRule {
```

Comment:
This looks ugly to me. If it doesn't fit into a single line, one solution could be to extract some conformances to extensions if possible. Something like this:

```swift
public struct EmptyMethodOverrideRule: SubstitutionCorrectableASTRule {
...
}

extension EmptyMethodOverrideRule: ConfigurationProviderRule, AutomaticTestableRule {}
```

If that's not possible or doesn't fit well, I'd rather indent it like this:

```swift
public struct EmptyMethodOverrideRule: SubstitutionCorrectableASTRule, 
        ConfigurationProviderRule, AutomaticTestableRule { // indent by 8 spaces
    public var configuration = SeverityConfiguration(.warning) // normal indent
```
or even like this:
```swift
public struct EmptyMethodOverrideRule: 
    SubstitutionCorrectableASTRule, 
    ConfigurationProviderRule, 
    AutomaticTestableRule 
{
    public var configuration = SeverityConfiguration(.warning) // normal indent
```
which I agree also looks weird though. 😅

Reply:
I really like the `extension X: Protocol` approach, but the SwiftLint default seems to be to have all protocols on line 1, so I went with that.